scala/scala-impl/src/org/jetbrains/plugins/scala/actions/NewPredefinedSbtFileAction.scala

```diff
@@ -0,0 +1,31 @@
+package org.jetbrains.plugins.scala.actions
+
+import com.intellij.ide.IdeView
+import com.intellij.ide.fileTemplates.actions.CreateFromTemplateActionBase
+import com.intellij.ide.fileTemplates.{FileTemplate, FileTemplateManager}
+import com.intellij.openapi.actionSystem.{CommonDataKeys, DataContext}
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.Messages
+import com.intellij.openapi.util.NlsActions
+import com.intellij.psi.PsiDirectory
+import org.jetbrains.plugins.scala.ScalaBundle
+import org.jetbrains.plugins.scala.extensions.PsiNamedElementExt
+
+import javax.swing.Icon
+
+abstract class NewPredefinedSbtFileAction(@NlsActions.ActionText title: String, @NlsActions.ActionDescription description: String, icon: Icon, fileName: String) extends CreateFromTemplateActionBase(title, description, icon) {
+
+  override def getTemplate(project: Project, dir: PsiDirectory): FileTemplate = FileTemplateManager.getDefaultInstance.getInternalTemplate(ScalaBundle.message("newclassorfile.menu.action.sbt.text"))
```

Comment:
`FileTemplateManager.getDefaultInstance.getInternalTemplate(ScalaBundle.message("newclassorfile.menu.action.sbt.text"))` should be in a new line. It is not readable in IDE